<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Proyecto AMN</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>241</width>
      <height>271</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>1</y>
       <width>110</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Imagen Test</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbImagen">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>241</width>
       <height>191</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btExaminarImagen">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>141</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Cargar Imagen Test</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>331</width>
      <height>271</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Imagen Encontrada</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbEncontrada">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>281</width>
       <height>241</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbImagenBusqueda">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>211</width>
       <height>161</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>661</width>
      <height>71</height>
     </rect>
    </property>
    <widget class="QLabel" name="lbBaseDatos">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>631</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ruta de los Archivos</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>441</width>
      <height>61</height>
     </rect>
    </property>
    <widget class="QPushButton" name="btBaseDatos">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>170</width>
       <height>27</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Ingrese la base de Imagenes pgm</string>
     </property>
     <property name="text">
      <string>Cargar Base Datos</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btCargar">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Entrenar el software</string>
     </property>
     <property name="text">
      <string>Entrenar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btBuscar">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Hacer el Test</string>
     </property>
     <property name="text">
      <string>Test</string>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>20</width>
      <height>271</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
